### ISSUE TITLE
InstanceNormalization scale and bias are represented as constant tensors

### ISSUE THREAD

**Issue by maksimgusarovsc** [onnx/onnx#1349]
I'm exporting a model with InstanceNormalization from Pytorch to ONNX. For some reason in ONNX InstanceNormalization's scale and bias parameters are represented as stand-alone **constant layers**, while all other parameters like BatchNormalization's scale, bias, mean, variance, etc. are represented as **initializers**. Can you guys comment on why there's such inconsistency?
